_build/templates/default/sass/_navbar.scss

```diff
@@ -38,7 +38,7 @@
   z-index: 20;
   display: flex;
   flex-direction: column;
-  padding: 0 10px;
+  padding: 0 5px;
```

Comment:
Is this PR for adding new screen width or for changing current? Why do you change default `#modx-navbar padding`, `.x-tab-panel-noborder margin` etc?

Reply:
@Ruslan-Aleev can you answer @mishantrop's question? :D

Reply:
I reduced the width of menu and tree by a few pixels, it is clear from commits.
Changed the title to make it clearer.

Reply:
But the PR title speaks of small screens and the changes are mostly for all widths. This is somehow misleading.